language/move-lang/src/cfgir/translate.rs

```diff
@@ -269,7 +269,7 @@ fn block_(context: &mut Context, cur_label: &mut Label, blocks: H::Block) -> Bas
     }
 
     macro_rules! loop_block {
-        (begin: $begin:expr, end: $end:expr, body: $body:expr, $block:expr) => {{
+        (begin: $begin:expr,end: $end:expr,body: $body:expr, $block:expr) => {{
```

Comment:
Is this a glitch in cargo fmt? It seems like those blank spaces should be present.

Reply:
Oof, yeah that didn't turn out great. I'll turn that flag off